Misc/Freetext_Airspace Bases.txt

```diff
@@ -48,6 +48,15 @@ N051.25.03.581:E001.48.54.239:Airspace Bases:FL105
 
 ;Cotswold CTA 14 FL75-FL195
 N051.35.33.209:W002.08.08.305:Airspace Bases:FL75
+;Cotswold CTA 15 C FL125-FL195
+N051.50.35.000:W002.39.35.000:Airspace Bases:*FL125/C
```

Comment:
Maybe *FL125*C to keep with other conventions?

Reply:
Have adjusted to *FL125C etc